include/ginkgo/core/base/matrix_data.hpp

```diff
@@ -484,14 +484,19 @@ struct matrix_data {
     /**
      * Sorts the nonzero vector so the values follow row-major order.
      */
-    void ensure_row_major_order()
+    void sort_row_major()
     {
         std::sort(
             begin(nonzeros), end(nonzeros), [](nonzero_type x, nonzero_type y) {
                 return std::tie(x.row, x.column) < std::tie(y.row, y.column);
             });
     }
 
+    [[deprecated("Use sort_row_major() instead")]] void ensure_row_major_order()
```

Comment:
there is still usage of this:
- this file line 263
- this file line 517
- test/factorization/cholesky_kernels.cpp line 314